rpcs3/Emu/System.cpp

```diff
@@ -1625,8 +1625,9 @@ void Emulator::Stop(bool restart)
 	cpu_thread::stop_all();
 	g_fxo->reset();
 
-	while (thread_ctrl::get_count())
+	while (u32 x = thread_ctrl::get_count())
 	{
+		sys_log.fatal("Waiting for %u threads...");
 		std::this_thread::sleep_for(10ms);
 	}
 
```

Comment:
There I think should be X passed to log.fatal.

Reply:
Thanks, also increased the timeout.